server/routes/overview/overviewController.ts

```diff
@@ -99,6 +101,31 @@ export default class OverviewController {
     const { prisonerSummary } = req.session
 
     const supportNeeds = await this.curiousService.getPrisonerSupportNeeds(prisonNumber, req.user.username)
+
+    // Loop through the healthAndSupport needs array and update the prison name for each need
+    await Promise.all(
+      supportNeeds.healthAndSupportNeeds.map(async supportNeed => {
+        const prison = await this.prisonService.getPrisonByPrisonId(supportNeed.prisonId, req.user.token)
```

Comment:
This call (and the one below for neurodiversities below) are using the wrong token
It works for you because your user token has some additional privs (including those to be able to call the prison-register-api), but not all users have these roles
Have a look at how this service is called by the timeline, but you want to be using the system token, not a user token